README.md

```diff
@@ -49,25 +56,23 @@ The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool
 
 ## Quick Start
 
-i. Install [`nextflow`](https://nf-co.re/usage/installation)
-
-ii. Install either [`Docker`](https://docs.docker.com/engine/installation/) or [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/) for full pipeline reproducibility (please only use [`Conda`](https://conda.io/miniconda.html) as a last resort; see [docs](https://nf-co.re/usage/configuration#basic-configuration-profiles))
-
-iii. Download the pipeline and test it on a minimal dataset with a single command
+1. Install [`nextflow`](https://nf-co.re/usage/installation)
```

Comment:
Will need to change some of this stuff in the template too 😅

Reply:
Yeah, I wondered that.. 🙄

Reply:
https://github.com/nf-core/tools/pull/621